src/components/FoodPantryDisplay/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { FoodPantry } from 'utils/types';
+import FoodPantryContainer from 'src/components/FoodPantryContainer/index';
+import { Pagination } from '@material-ui/lab';
+
+interface Props {
+  pantries: FoodPantry[];
+}
+
+interface CardDisplayState {
+  offset: number;
+}
+
+class FoodPantryDisplay extends Component<Props, CardDisplayState> {
+  // Set default offset to 0
+  constructor(props: Props) {
+    super(props);
+    this.state = { offset: 0 };
+  }
+
+  // setState on pagination change
+  onPageChange = (event: React.ChangeEvent<unknown>, page: number) => {
+    this.setState({
+      offset: page - 1,
+    });
+  };
+
+  // Get correct pantries to display
+  getPantriesToDisplay = () => {
+    const { pantries } = this.props;
+    const { offset } = this.state;
+
+    // Syntax can be changed depending on what we decide on
+    return (pantries.length < 3 && pantries) || pantries.slice(offset, offset + 3);
+  };
+
+  renderPagination = () => {
+    const { pantries } = this.props;
+    const { offset } = this.state;
+    const pageCount: number = pantries.length - 3 > 0 ? pantries.length - 3 : 1;
```

Comment:
No need for explicit type annotation in this instance
```suggestion
    const pageCount = pantries.length - 3 > 0 ? pantries.length - 3 : 1;
```